src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin calendar-day {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  width: $day-width;
+  height: $day-width;
+  box-sizing: border-box;
+  border: 1px solid #000;
+  background-color: $color-grey;
+
+  transition: all $animation-duration ease-in-out;
```

Comment:
Don't use transition all, specify to which properties it should apply
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x